plugins/tiddlywiki/multiwikiserver/config/MultiWikiServerAllowAnnonymousReads.tid

```diff
@@ -0,0 +1,2 @@
+title: $:/config/MultiWikiServer/MultiWikiServerAllowAnnonymousReads
```

Comment:
In TW if config values are undefined by default, the config tiddlers do not exist. 
Also the second `Multiwikiserver`  information is redundant. So imo it can be removed.

Reply:
Also boolean information is stored as `yes` or `no`, which is easier to understand for non programmers

Reply:
I set the initial value to "undefined" because when it is undefined, then the system will prompt the user to then specify a yes or no value. The other comments are all fixed.